backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/services/GeneralViewEdgeToolSwitch.java

```diff
@@ -300,7 +301,7 @@ private EdgeTool createAddAsNestedPartEdgeTool(List<NodeDescription> targetEleme
                 .expression(AQLConstants.AQL + EdgeDescription.SEMANTIC_EDGE_SOURCE + ".addAsNestedPart(" + EdgeDescription.SEMANTIC_EDGE_TARGET + ")");
 
         return builder
-                .name("Add Part Usage as nested " + SysmlPackage.eINSTANCE.getPartUsage().getName())
+                .name(DescriptionNameGenerator.getCreationToolName("Add Part Usage as nested ", SysmlPackage.eINSTANCE.getPartUsage()))
```

Comment:
May be also remove `Usage` here in `Add Part Usage as nested ` -> `Add Part as nested `

Reply:
done